[2023-04-05T04:29:58.536+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT.object_transformation.sql manual__2023-04-05T04:29:55.407763+00:00 [queued]>
[2023-04-05T04:29:58.548+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT.object_transformation.sql manual__2023-04-05T04:29:55.407763+00:00 [queued]>
[2023-04-05T04:29:58.549+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-05T04:29:58.549+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-05T04:29:58.549+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-05T04:29:58.567+0000] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): object_transformation.sql> on 2023-04-05 04:29:55.407763+00:00
[2023-04-05T04:29:58.573+0000] {standard_task_runner.py:55} INFO - Started process 593 to run task
[2023-04-05T04:29:58.577+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ELT', 'object_transformation.sql', 'manual__2023-04-05T04:29:55.407763+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmplay3j213']
[2023-04-05T04:29:58.578+0000] {standard_task_runner.py:83} INFO - Job 69: Subtask object_transformation.sql
[2023-04-05T04:29:58.658+0000] {task_command.py:389} INFO - Running <TaskInstance: ELT.object_transformation.sql manual__2023-04-05T04:29:55.407763+00:00 [running]> on host 08fcf180c6cc
[2023-04-05T04:29:58.743+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=200300600@ucaribe.edu.mx
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ELT
AIRFLOW_CTX_TASK_ID=object_transformation.sql
AIRFLOW_CTX_EXECUTION_DATE=2023-04-05T04:29:55.407763+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-05T04:29:55.407763+00:00
[2023-04-05T04:29:58.749+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-04-05T04:29:58.750+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'dbt ls']
[2023-04-05T04:29:58.769+0000] {subprocess.py:86} INFO - Output:
[2023-04-05T04:30:01.083+0000] {subprocess.py:93} INFO - [0m04:30:01  Encountered an error:
[2023-04-05T04:30:01.084+0000] {subprocess.py:93} INFO - Runtime Error
[2023-04-05T04:30:01.084+0000] {subprocess.py:93} INFO -   fatal: Not a dbt project (or any of the parent directories). Missing dbt_project.yml file
[2023-04-05T04:30:01.335+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-04-05T04:30:01.347+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-04-05T04:30:01.351+0000] {taskinstance.py:1332} INFO - Marking task as FAILED. dag_id=ELT, task_id=object_transformation.sql, execution_date=20230405T042955, start_date=20230405T042958, end_date=20230405T043001
[2023-04-05T04:30:01.362+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 69 for task object_transformation.sql (Bash command failed. The command returned a non-zero exit code 2.; 593)
[2023-04-05T04:30:01.398+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-04-05T04:30:01.417+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
